.github/workflows/docker-build-push-sentry.yml

```
@@ -36,6 +36,10 @@ jobs:
       - name: Install Python dependencies
         run: pip install -r requirements.txt
 
+      - name: Inject Version into Entrypoint
```

actionlint (1.7.4)

step must run script with "run" section or run action with "uses" section

(syntax-check)